draft-ietf-sidrops-aspa-verification.xml

```diff
@@ -212,20 +209,17 @@
      </t>
   </section>
 
-    <section title="Autonomous System Provider Authorization" anchor="ASPA">
+    <section title="Autonomous System Provider Authorization Record" anchor="ASPA">
       <t>
         An ASPA record is a digitally signed object that binds a set of Provider AS numbers to a Customer AS (CAS) number (in terms of BGP announcements) and is signed by the CAS <xref target="I-D.ietf-sidrops-aspa-profile"/>.
         The ASPA attests that the CAS indicated a Set of Provider ASes (SPAS), which applies only to the IPv4 and IPv6 address families (i.e., AFI = 1 and AFI = 2) and only to Network Layer Reachability Information used for unicast forwarding (SAFI = 1).
         The definition of Provider AS is given in Section 1 of the ASPA profile object document <xref target="I-D.ietf-sidrops-aspa-profile"/>.
-        A function of a Provider AS is to propagate a CAS's route announcements onward, i.e., to the Provider's upstream providers, lateral peers, or customers.
```

Comment:
-> explains what's the purpose of a provider in a section that is dedicated to the ASPA Record,